source/fundamentals/builders.txt

```diff
@@ -364,6 +364,43 @@ as a ``BsonDocument`` to the `AppendStage() method
 To learn more about the Aggregation Pipeline, see the
 :manual:`Aggregation Pipeline </core/aggregation-pipeline/>` server manual page.
 
+.. _csharp-builders-set:
+
+Add Fields to Collection Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can add new fields to existing documents in a collection 
```

Comment:
This might be a bit misleading because the documents in the collection are NOT modified.

The new fields are only added to the documents going through the *pipeline*, the original documents that are input to the pipeline are unchanged.